0-ad-history-mod-package/simulation/templates/units/han/hero_han_xin.xml

```diff
@@ -8,6 +8,16 @@
     <Civ>han</Civ>
     <SpecificName>Hán Xìn</SpecificName>
     <Icon>units/han/hero_han_xin.png</Icon>
+    <History>
+        Han Xin grew up in bitter poverty. He was so poor, in fact, that he constantly struggled to get the necessary food and often had to rely on others. For that, he was despised by his town's residents. He even lost his mother very early in life and did not have the financial means to properly bury her. So, when rebellions broke out throughout the empire, Han Xin saw his chance and joined the rebel leader Xiang Yu.
+        Xiang Yu was a warlord renown for his bravery and considered the greatest warrior of his time. Together with his elite soldiers, he was regarded as almost invincible, even having defeated armies twenty times the size of his own.
+        It's said that Han Xin often made tactical suggestions to higher officers but felt not taken seriously and ignored. So, after Xiang Yu had defeated the Qin dynasty and split China into eighteen kingdoms, Han Xin decided to move over to the kingdom of Han under Liu Bang. Maybe because Liu Bang came, unlike other warlords, from destitue backgrounds as well. However, it didn't work out for him there either: for a small crime, he was put in jail and barely escaped execution. Only when he was about to desert again and find his luck somewhere else did some of Liu Bang's advisors, who appeared to be very impressed by the young soldier, gave him a chance and led him to Liu Bang himself. Han Xin was then, surprisingly, appointed chief commander of Liu Bang's armies.
```

Comment:
```suggestion
        It's said that Han Xin often made tactical suggestions to higher officers, but his advice was not taken seriously and often ignored. So, after Xiang Yu had defeated the Qin dynasty and split China into eighteen kingdoms, Han Xin decided to move over to the kingdom of Han under Liu Bang. Maybe because Liu Bang came, unlike other warlords, from destitute backgrounds as well. However, it didn't work out for him there either: for a small crime, he was put in jail and barely escaped execution. Only when he was about to desert again and find his luck somewhere else did some of Liu Bang's advisors, who appeared to be very impressed by the young soldier, gave him a chance and led him to Liu Bang himself. Han Xin was then, surprisingly, appointed chief commander of Liu Bang's armies.
```